Assets/Scripts/Controllers/GameController.cs

```diff
@@ -0,0 +1,76 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+
+using ProjectPorcupine.Localization;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameController : MonoBehaviour
+{
+    public KeyboardManager KeyboardManager;
```

Comment:
You probably don't even need this. Unless we start removing some static classes.